File: Easy/longest_word.py
data = ['some line with text', 'another line']


#import sys
#test_cases = open(sys.argv[1], 'r')
#data = test_cases.readlines()
for test in data:
	test = test.rstrip()
	l = test.split(" ")
	longest = ''
	max_len = 0
	for i in l:
		length = len(i)
		if length > max_len:
			max_len = length
			longest = i
	print longest

#test_cases.close()

File: Easy/longest_word.r
test.cases <- c("some line with text", "another line")

#args <- commandArgs(trailingOnly=TRUE)
#test.cases <- strsplit(readLines(args[[1]], warn=FALSE), '\n')
cat(sapply(test.cases, function(test) {
	if (length(test) > 0) {
		words <- strsplit(test, " ")[[1]]
		len <- words
		len <- sapply(len, function(x) {nchar(x)})
		M <- max(len)
		names(len[len == M])[1]
	}
}), sep = "\n")